src/components/ProjectsPage/index.tsx

```diff
@@ -0,0 +1,158 @@
+import ProjectCard from '@components/ProjectCard';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+import styles from './ProjectsPage.module.css';
+import { useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { projectList } from './ProjectsPageData';
+import { useTranslation } from 'react-i18next';
+
+function ProjectsPage() {
+    const filterButtons = [
+        'fixed-price',
+        'dedicated',
+        'active',
+        'inactive',
+        'cancelled',
+    ];
```

Comment:
Everything inside of the function component's body re-binds on each re-render, so it's best to keep things that are constant outside of the function body. It can be within the same file, but for readability it might be best to move it inside src/constants/projects.ts